src/OrchardCore.Modules/OrchardCore.Workflows/Trimming/Startup.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.BackgroundTasks;
+using OrchardCore.DisplayManagement.Handlers;
+using OrchardCore.Environment.Shell.Configuration;
+using OrchardCore.Modules;
+using OrchardCore.Navigation;
+using OrchardCore.Security.Permissions;
+using OrchardCore.Settings;
+using OrchardCore.Workflows.Trimming.Drivers;
+using OrchardCore.Workflows.Trimming.Services;
+
+namespace OrchardCore.Workflows.Trimming;
+
+[Feature("OrchardCore.Workflows")]
+public sealed class Startup : StartupBase
+{
+    private readonly IShellConfiguration _shellConfiguration;
+
+    public Startup(IShellConfiguration shellConfiguration)
+    {
+        _shellConfiguration = shellConfiguration;
+    }
+
+    public override void ConfigureServices(IServiceCollection services)
+    {
+        services.AddScoped<IPermissionProvider, Permissions>();
+        services.AddScoped<IWorkflowTrimmingManager, WorkflowTrimmingManager>();
+        services.AddSingleton<IBackgroundTask, WorkflowTrimmingBackgroundTask>();
+        services.AddScoped<IDisplayDriver<ISite>, WorkflowTrimmingDisplayDriver>();
+        services.AddScoped<INavigationProvider, AdminMenu>();
+
+        services.Configure<WorkflowTrimmingOptions>(_shellConfiguration.GetSection("OrchardCore_Workflows").GetSection("Trimming"));
```

Comment:
I think you can use one GetSection and separate the sections by `:`

Reply:
Yes, but that'll only work under Windows, while double underscores will only work under Unix platforms. This is safe for every platform.

Reply:
I think the conversion is done for you automatically even when you use `:`.